### DISCUSSION TITLE
Climatology dataset sources

### DISCUSSION THREAD

**Discussion by jwraby** [dtcenter/METplus#2400] [File I/O] [answered]
I appreciated the METplus Overview presentation given by Tara. The topics she covered brought a couple of questions to mind. Regards doing comparative verification using the climatology data settings in the Point-Stat and Grid-Stat configuration files, what are some widely accepted sources of MET-compatible datasets which are appropriate for this? I was also interesting to hear about the METplus use cases during the training. Reviewing those for the S2S topic, I saw the various types of data files listed (forecast and observations) and wondered if you could provide the sources for those files?

**Comment by j-opatz** [comment, marked as answer]
Hi John,

And thank you for your question. As a broad answer, [the MET User's Guide defines the input formats](https://met.readthedocs.io/en/latest/Users_Guide/data_io.html#input-data-formats) that METplus can accept natively. If you find a data source and it conforms to these guidelines, then it most likely (barring any unexpected issues) can be ingested with METplus without any additional preprocessing.

For more examples of datasets that have been used, you can (and already have) check out the [METplus User's Guide for all of the use cases that use operational datasets](https://metplus.readthedocs.io/en/latest/generated/model_applications/index.html#model-applications)  and how they were utilized. A small but growing source for datasets used with METplus is located [here, in the Verification and Datasets Guide](https://metplus.readthedocs.io/en/latest/Verification_Datasets/index.html#verification-datasets-guide). This guide is by no means exhaustive of all of the datasets that could be used natively with METplus (it still needs to be updated to list all of the datasets currently in the use cases), but it might be a good starting point for you.

This is speaking purely from my experience on the METplus team, but very often we receive the datasets for use cases from the centers and organizations that are helping us create the use case. For example, during my work on a recent project with the Climate Prediction Center (CPC) we ended up create 2 separate use cases. For those use cases CPC provided all of the datasets, including the GHCNCAMS for observations and CFSv2 data for forecasts. 

If you ever want to use the same data that is being used in the use cases, you can find them in tarballs, separated by each category (s2s, marine and cryosphere, etc.) in the associated release directory. [Here is the one for METplus version 5.1](https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/v5.1/), and [this is for the upcoming METplus release 6.0](https://dtcenter.ucar.edu/dfiles/code/METplus/METplus_Data/v6.0/) (note that this one will be updated as betas continue to introduce new use cases).

**Reply by jwraby** [reply]
John - Thank you for compiling all the useful information which, from a METplus perspective, applies directly to addressing my question. Again, you have saved me considerable time in gathering this information on my own. Your expertise in analyzing my needs is greatly appreciated. I will use your suggestions and information to direct my search for climatology data in a way which will produce results for this project much quicker.
